tests/resources/app/config/CacheBox.cfc

```diff
@@ -0,0 +1,50 @@
+﻿component{
```

Comment:
Hiding the testing integration app under `resources` is too much for me. Can we put this under `tests/app` instead.